orttraining/orttraining/training_ops/cpu/cpu_training_kernels.cc

```diff
@@ -88,7 +76,10 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1,
 
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kMSDomain, 1, float_float, ReduceAllL2);
 
+#ifdef ENABLE_TRAINING
```

Comment:
Let's add a comment here to explain the following kernels will no be included in 
--enable_training_ops build